src/libraries/System.Diagnostics.DiagnosticSource/ref/System.Diagnostics.DiagnosticSourceActivity.cs

```diff
@@ -169,6 +171,14 @@ public sealed class ActivitySource : IDisposable
         public static void AddActivityListener(System.Diagnostics.ActivityListener listener) { throw null; }
         public void Dispose() { throw null; }
     }
+    public class ActivitySourceOptions
+    {
+        public ActivitySourceOptions(string name) { throw null; }
+        public string Name { get { throw null; } set { throw null; } }
```

Comment:
We don't need these throw nulls in the setters, do we?

Reply:
Why not? in ref code we do that all the time.

Reply:
Not for setters. We do it when the implementation needs to return something or set an out parameter; setters don't, e.g.
https://github.com/dotnet/runtime/blob/577ebdc00cc368c8d6ec5b4fd5f6df8b1e1b651a/src/libraries/System.Diagnostics.DiagnosticSource/ref/System.Diagnostics.DiagnosticSourceActivity.cs#L22
I'm not sure why some of the setters in this file do it... I'm guessing they were written by hand rather than by genapi.

Reply:
Ok, I'll fix that.